src/converter.py

```diff
@@ -0,0 +1,173 @@
+import datetime
+import json
+
+def convert_tes_to_wrroc(tes_data):
+    try:
+        if not tes_data or not isinstance(tes_data, dict):
+            raise ValueError("Invalid TES data provided")
+
+        if not tes_data.get("executors"):
+            raise ValueError("TES data missing 'executors' list or 'executors' list is empty")
```

Comment:
This is insufficient to validate the TES response data. It's not like _any_ dict with a `executors` key would be a valid TES response. You need to define models for TES and WES, e.g., with Pydantic. Please discuss this with @JaeAeich - he is already writing models for TES (you could reuse them and add the models for WES).